webextensions/api/declarativeNetRequest.json

```diff
@@ -0,0 +1,1656 @@
+            }
+          }
+        },
+        "onRuleMatchedDebug": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/declarativeNetRequest/onRuleMatchedDebug",
+            "support": {
+              "chrome": {
+                "version_added": "84",
+                "notes": "Available only to unpacked extensions."
+              },
+              "edge": "mirror",
+              "firefox": {
+                "version_added": false
+              },
+              "firefox_android": "mirror",
+              "opera": "mirror",
+              "safari": {
+                "version_added": "15"
+              },
+              "safari_ios": "mirror"
+            }
+          },
+          "request": {
+            "__compat": {
+              "support": {
+                "chrome": {
+                  "version_added": "84"
+                },
+                "edge": "mirror",
+                "firefox": {
+                  "version_added": false
+                },
+                "firefox_android": "mirror",
+                "opera": "mirror",
+                "safari": {
+                  "version_added": "15"
+                },
+                "safari_ios": "mirror"
+              }
+            },
+            "documentId": {
+              "__compat": {
+                "support": {
+                  "chrome": {
+                    "version_added": "106"
+                  },
+                  "edge": "mirror",
+                  "firefox": {
+                    "version_added": false
+                  },
+                  "firefox_android": "mirror",
+                  "opera": "mirror",
+                  "safari": {
+                    "version_added": false
+                  },
+                  "safari_ios": "mirror"
+                }
+              }
+            },
+            "documentLifecycle": {
+              "__compat": {
+                "support": {
+                  "chrome": {
+                    "version_added": "106"
+                  },
+                  "edge": "mirror",
+                  "firefox": {
+                    "version_added": false
+                  },
+                  "firefox_android": "mirror",
+                  "opera": "mirror",
+                  "safari": {
+                    "version_added": false
+                  },
+                  "safari_ios": "mirror"
+                }
+              }
+            },
+            "frameId": {
+              "__compat": {
+                "support": {
+                  "chrome": {
+                    "version_added": "84"
+                  },
+                  "edge": "mirror",
+                  "firefox": {
+                    "version_added": false
+                  },
+                  "firefox_android": "mirror",
+                  "opera": "mirror",
+                  "safari": {
+                    "version_added": "15"
```

Comment:
`onRuleMatchedDebug` and related are not supported by Safari.

Reply:
```suggestion
                    "version_added": false
```